packages/ui/src/common/api/schemas/schema.graphql

```diff
@@ -841,13 +846,13 @@ type Bounty implements BaseGraphQLObject {
   stage: BountyStage!
 
   """
-  Total amount available for the reward
+  Total amount currently available for the reward
   """
   totalFunding: BigInt!
```

Comment:
@RafalPloszka can this `totalFunding` be used to check the unwithdrawn fund value during the withdrawal periods ?

Reply:
Sure, nice